docs/pages/core-concepts.mdx

```diff
@@ -200,8 +209,9 @@ authentication works in Teleport](./access-controls/sso.mdx).
 
 ### Authentication connector
 
-An authentication connector is a **configuration resource** that allows users to
-authenticate to Teleport via a Single Sign-On (SSO) solution. 
+An authentication connector (also known as Auth connector) is a 
```

Comment:
```suggestion
An authentication connector (also known as an auth connector) is a 
```

Reply:
The other possibility is to update the rest of the docs so we only say "authentication connector":

```bash
grep -RhiEo "auth(entication)? connector" docs/pages | s
ort | uniq -c
   1 Auth Connector
   1 Auth connector
   1 Authentication Connector
   4 Authentication connector
  19 auth connector
  37 authentication connector
```